unified-prototype/unified-plugin/plugin-cpp/build.gradle.kts

```
@@ -0,0 +1,30 @@
+plugins {
+    `kotlin-dsl`
+    id("build-logic.publishing")
+}
+
+description = "Implements the declarative C++ DSL prototype"
+
+dependencies {
+    implementation(project(":plugin-common"))
+}
+
+gradlePlugin {
+    plugins {
+        create("cpp-library") {
+            id = "org.gradle.experimental.cpp-library"
+            implementationClass = "org.gradle.api.experimental.cpp.StandaloneCppLibraryPlugin"
+            tags = setOf("declarative-gradle")
+        }
+        create("cpp-application") {
+            id = "org.gradle.experimental.cpp-application"
+            implementationClass = "org.gradle.api.experimental.cpp.StandaloneCppApplicationPlugin"
+            tags = setOf("declarative-gradle")
+        }
+        create("swift-ecosystem") {
```

❓ Looks like a cut/paste oversight...
```suggestion
        create("cpp-ecosystem") {
```